.ci/build-mac.sh

```diff
@@ -53,15 +75,17 @@ mkdir build && cd build || exit 1
     -DLLVM_INCLUDE_DOCS=OFF -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DLLVM_INCLUDE_TOOLS=OFF \
     -DLLVM_INCLUDE_UTILS=OFF -DLLVM_USE_PERF=OFF -DLLVM_ENABLE_Z3_SOLVER=OFF \
     -DUSE_NATIVE_INSTRUCTIONS=OFF \
-    -DUSE_SYSTEM_MVK=OFF \
+    -DUSE_SYSTEM_MVK=ON \
```

Comment:
Won't this override our moltenvk submodule version?

Reply:
Are there important changes there? The goal of this is to use prebuilt libs to speed up the process of compiling

Reply:
What we're worried about is that system MoltenVK version silently changes in CI when we push other changes to master and that MoltenVk update causes regressions. We'll then spend a long time going through the changes on our side and how they could have caused the regression when they had nothing to do with it.

But kd-11 can decide what to do about this.

Reply:
Hmm ok i can revert back to it , it will just add to compile time

Reply:
Can we specify the version of moltenvk installed by brew? We don't need to compile it ourselves as we don't ship any patches with it. We just need to have a guarantee that we're all on the same version that has been validated by the team.

Reply:
Sure which version do we want